src/elements/Payment.vue

```diff
@@ -129,19 +129,31 @@ export default {
       try {
         this.$emit('loading', true);
         event.preventDefault();
-        const { error } = await this.stripe.confirmPayment({
+        const response = await this.stripe.confirmPayment({
```

Comment:
Destructure the `error` and `paymentIntent` instead of using the `response` directly.